kura/org.eclipse.kura.net.admin/src/main/java/org/eclipse/kura/net/admin/modem/zte/me3630/ZteMe3630ConfigGenerator.java

```diff
@@ -72,6 +72,7 @@ public PppPeer getPppPeer(String deviceId, ModemConfig modemConfig, String logFi
 
     @Override
     public ModemXchangeScript getConnectScript(ModemConfig modemConfig) {
+        // PDP context is fixed to 1 for this modem
```

Comment:
Maybe this needs a separate PR as this regards the Quectel modems

Reply:
OK.